src/pages/encrypt/index.js

```diff
@@ -29,7 +29,8 @@ module.exports = React.createClass({
     var coralScript = null;
     if (this.props.buyersGuide) {
       googleFonts += "|Playfair+Display:400,400i,700,900";
-      googleFonts += "|Open+Sans:300,600";
+      googleFonts += "|Open+Sans:300,400,600";
+      googleFonts += "|Zilla+Slab:300,600";
```

Comment:
Odd I thought this happened.

Reply:
No I made some more changes, this time to the item reviews.